subsys/fs/nvs/nvs.c

```diff
@@ -16,6 +16,75 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(fs_nvs, CONFIG_NVS_LOG_LEVEL);
 
+static int nvs_prev_ate(struct nvs_fs *fs, uint32_t *addr, struct nvs_ate *ate);
+static int nvs_ate_valid(struct nvs_fs *fs, const struct nvs_ate *entry);
+
+#ifdef CONFIG_NVS_LOOKUP_CACHE
+
+#define NVS_LOOKUP_CACHE_EMPTY 0
+
```

Comment:
`NVS_LOOKUP_CACHE_NO_ADDRESS` ? In theory a ATE could be found at address 0, in practice this case doesn't matter because it would be a delete. Maybe this could be added as a remark for future questions.

Reply:
Changed both the constant name and value.